src/posts/posts.controller.ts

```diff
@@ -78,10 +77,10 @@ export class PostsController {
     @ApiQuery({ name: 'limit', type: 'number', required: false })
     @ApiQuery({ name: 'order', enum: Order, required: false, example: 'ASC' })
     @ApiQuery({ name: 'condition', enum: OrderCondition, required: false, example: 'ID' })
-    @Get()
+    @Get('/')
     async findAll(
-        @Query('page', new DefaultValuePipe(1), ParseIntPipe) page: number = 1,
-        @Query('limit', new DefaultValuePipe(10), ParseIntPipe) limit: number = 10,
+        @Query('page', new DefaultValuePipe(1), ParseIntPipe) page = 1,
```

Comment:
Please use constant variables instead of hardcoded values.
something like this: `PAGE_DEFAULT_VALUE` instead of `1`.

Also, please check if this code is needed:
`new DefaultValuePipe(1), ParseIntPipe`